App Sec OOTB Rules (#12934)
DataDog/documentation: content/en/security_platform/default_rules/_index.md

disable_sidebar: true
---

> Datadog provides out-of-the-box (OOTB) detection rules to flag attacker techniques and potential misconfigurations so you can immediately take steps to remediate. Datadog continuously develops new default rules, which are automatically imported into your account or with the Agent. For more information, see the Detection Rules documentation.

Reviewer: Agent + library (AppSec)

> Datadog provides out-of-the-box (OOTB) detection rules to flag attacker techniques and potential misconfigurations so you can immediately take steps to remediate. Datadog continuously develops new default rules, which are automatically imported into your account or with the Agent. For more information, see the Detection Rules documentation.

Author: ```suggestion
Datadog provides out-of-the-box (OOTB) detection rules to flag attacker techniques and potential misconfigurations so you can immediately take steps to remediate. Datadog continuously develops new default rules, which are automatically imported into your account, your Application Security library, and with the Agent, depending on your configuration. For more information, see the Detection Rules documentation.
```

> Datadog provides out-of-the-box (OOTB) detection rules to flag attacker techniques and potential misconfigurations so you can immediately take steps to remediate. Datadog continuously develops new default rules, which are automatically imported into your account, your Application Security library, and with the Agent, depending on your configuration. For more information, see the Detection Rules documentation.

Reviewer 2: ```suggestion
Datadog provides out-of-the-box (OOTB) detection rules to flag attacker techniques and potential misconfigurations so you can immediately take steps to remediate. Datadog continuously develops new default rules, which are automatically imported into your account, your Application Security library, and the Agent, depending on your configuration. For more information, see the Detection Rules documentation.
```

Reviewer 2: Also, is there a link for "Detection Rules documentation"?

Author: Why yes, there is! ✨  Added.

> Click on the buttons below to filter by different parts of the Datadog Security Platform. OOTB rules are available for [Cloud SIEM][1], [Posture Management][2], which is divided into cloud or infrastructure configuration, [Workload Security][3], and Application Security.

Reviewer: Miss link to AppSec

Author: Intentionally removed as we are publishing this page first. Will add the link once Kari's branch is published with the docs.

[1]: /security_platform/cloud_siem/
[2]: /security_platform/cspm/

> [3]: /security_platform/cloud_workload_security/

Author: ```suggestion
[3]: /security_platform/cloud_workload_security/
[4]: /security_platform/detection_rules/
```
